library/src/tokenize/BasisTheoryTokenize.ts

```diff
@@ -0,0 +1,55 @@
+import type { AxiosTransformer } from 'axios';
```

Comment:
Does `Tokenize` have to be its own client? This differentiates from the .NET SDK where all token operations are in the same TokenClient class.